providers/fcm/src/lib/fcm.provider.spec.ts

```diff
@@ -257,3 +277,49 @@ test('should trigger fcm data for android with priority option', async () => {
     },
   });
 });
+
+test('should clean the payload for the FCM data message', async () => {
```

Comment:
test that makes sure that the FCM data message data field is `Record<string, string>`